README.md

```diff
@@ -52,14 +52,14 @@ Zudoku is quick to implement, easy to configure and is highly composable with se
 
 ## ⚙️ Installation
 
-You can use the CLI to generate a new project or use it standalone as a React component.
+You can use the CLI to generate a new project or use it standalone via CDN as a React component.
 
 ### ⚡️ Quick start
 
-Fire up your new API docs from out getting started template:
+Fire up your new API docs using the command line generator:
 
 ```
-npm create zudoku-app@latest
+npm create-zudoku-app@latest
```

Comment:
```suggestion
npm create zudoku-app@latest
```

This is the correct way to do it. For your change to work it'd need to be `npx create-zudoku-app@latest`. It's confusing